cfme/tests/configure/test_register_appliance.py

```diff
@@ -174,6 +175,20 @@ def test_rh_registration(appliance, request, reg_method, reg_data, proxy_url, pr
     request.addfinalizer(appliance.unregister)
 
 
+def test_rhsm_registration_check_repo_names(temp_appliance_preconfig_funcscope, soft_assert):
+    """ Checks default repo on a fresh appliance and checks """
```

Comment:
Would you change "Checks default repo" to "Checks default rpm repo".   That wil make it clearer what type of repo it is checking.